PocketKit/Sources/PocketKit/DataController.swift

```diff
@@ -6,16 +6,17 @@
 import Foundation
 @preconcurrency import SwiftData
 import SharedPocketKit
+import Sync
 
 public struct DataController: Sendable {
-    public func makeModelContainer(groupID: String) -> ModelContainer {
+    public func makeModelContainer() -> ModelContainer {
```

Comment:
the construction of `ModelContainer` was moved to `PocketKit` so there's no need to inject a `groupID` externally